packages/sdk38/src/cosmosclient.spec.ts

```diff
@@ -193,7 +193,9 @@ describe("CosmosClient", () => {
   describe("postTx", () => {
     it("works", async () => {
       pendingWithoutWasmd();
-      const pen = await Secp256k1Pen.fromMnemonic(faucet.mnemonic);
+      const wallet = await Secp256k1OfflineWallet.fromMnemonic(faucet.mnemonic);
+      const accounts = await wallet.getAccounts();
+      const { address: walletAddress } = accounts[0];
```

Comment:
Ever tried nested array and object destruction?

```ts
const [{ address: walletAddress }] = await wallet.getAccounts();
```

No idea if that works and is still readable.

Reply:
I like this and it works. :)